htdocs/stc/media.css

```diff
@@ -69,7 +69,7 @@
 
 .embed {padding: 0 1em;}
 
-@media screen and (min-width: 64em) {
+@media screen and (min-width: 64.01em) {
```

Comment:
I think you also need to apply a similar adjustment to the labels that appear after uploading a new file - those are still floating above and to the right at widths less than 960 but more than 600.